ci-operator/step-registry/acm/fetch-managed-clusters/acm-fetch-managed-clusters-commands.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+# Set the dynamic vars based on provisioned hub cluster.
+HUB_OCP_API_URL=$(oc whoami --show-server)
+export HUB_OCP_API_URL
```

Comment:
Why not just do `export HUB_OCP_API_URL=$(oc whoami --show-server)` ?

Reply:
Ah this was failing shellcheck

```
In ci-operator/step-registry/acm/fetch-managed-clusters/acm-fetch-managed-clusters-commands.sh line 9:
export HUB_OCP_API_URL=$(oc whoami --show-server)
       ^-------------^ SC2155 (warning): Declare and assign separately to avoid masking return values.
```

Reply:
Ah that makes sense, disregard the rest of the comments I made on this same thing.